common/os/os.cc

```diff
@@ -22,7 +21,7 @@ void *Joinable::trampoline(void *ptr) {
 
 unique_ptr<Joinable> runInAThread(string_view threadName, function<void()> function, optional<int> bindToCore) {
 #ifdef EMSCRIPTEN
-    sorbet::Exception::raise("Creating threads in unsupported in EMSCRIPTEN");
+    throw logic_error("Creating threads in unsupported in EMSCRIPTEN");
```

Comment:
These exception kinds are caught in main but maybe it’s fine to ignore that?

Reply:
Unfortunately they're not. :(  I can take a note to either catch them or do a fine-grained separation of things; it may be easier to just catch them.